Error: Webview API guide: Unhandled `undefined` value in `columnToShowIn` passed to reveal() method

The first code block in the [Visibility and Moving](https://code.visualstudio.com/api/extension-guides/webview#visibility-and-moving) section showing an example usage of the `reveal()` function contains the following error:
```
src/extension.ts:31:11 - error TS2345: Argument of type 'ViewColumn | undefined' is not assignable to parameter of type 'ViewColumn | { readonly viewColumn: ViewColumn; readonly preserveFocus?: boolean | undefined; }'.
  Type 'undefined' is not assignable to type 'ViewColumn | { readonly viewColumn: ViewColumn; readonly preserveFocus?: boolean | undefined; }'.

           columnToShowIn,
           ~~~~~~~~~~~~~~

[2:59:28 AM] Found 1 error. Watching for file changes.
```

--- a/api/extension-guides/webview.md
+++ b/api/extension-guides/webview.md
@@ -313,7 +313,7 @@ Let's update our extension to only allow a single webview to exist at a time. If
 
 ```ts
 export function activate(context: vscode.ExtensionContext) {
-  // Track currently webview panel
+  // Track the current panel with a webview
   let currentPanel: vscode.WebviewPanel | undefined = undefined;
 
   context.subscriptions.push(
@@ -330,7 +330,7 @@ export function activate(context: vscode.ExtensionContext) {
         currentPanel = vscode.window.createWebviewPanel(
           'catCoding',
           'Cat Coding',
-          columnToShowIn,
+          columnToShowIn || vscode.ViewColumn.One,
           {}
         );
         currentPanel.webview.html = getWebviewContent('Coding Cat');

